cypress.json

```
@@ -0,0 +1,7 @@
+{
+  "projectId": "slate-react",
```

Is slate react or slate the appropriate name here?
```suggestion
  "projectId": "slate",
```